packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -101,6 +101,22 @@
                     opacity: 0;
                 }
             }
+
+            #dt_purchase_vanillalongcall_button,
+            #dt_purchase_vanillalongput_button {
+                @include desktop {
+                    .btn-purchase__info--left {
+                        width: 80%;
```

Comment:
percentage is subject to the viewport size, make sure it works well in different screens.